src/util.js

```diff
@@ -55,8 +55,14 @@ export function getMetadata(metaFileContent, additionalGrantList) {
     .map(line => {
       if (!line.startsWith('// ')) return;
       line = line.slice(3).trim();
-      const i = line.indexOf(' ');
-      if (i < 0) return;
+      const i = line.search(/\s/);
```

Comment:
How about making it more general so we don't need to list keys without values?

```js
const matches = line.match(/^(\S+)(\s.*)?$/);
if (!matches) return;
const key = matches[1];
const value = (matches[2] || '').trim();
```

Reply:
That seems even better.